pkg/mermaid/endpointanalysisdiagram/endpointanalysisdiagram.go

```diff
@@ -84,7 +85,7 @@ func externalLinksContain(i []externalLink, ip externalLink) bool {
 	return false
 }
 
-//Replaces certain characters in the string suitable for mermaid
+//cleanString replaces certain characters in the string suitable for mermaid
```

Comment:
I reckon this should probably be in a single shared package rather than duplicated?

Reply:
Addressed